activerecord/lib/active_record/connection_adapters/mysql/quoting.rb

```diff
@@ -8,6 +8,8 @@ module MySQL
       module Quoting # :nodoc:
         def quote_bound_value(value)
           case value
+          when Rational
+            quote(value.to_f.to_s)
```

Comment:
Couldn't we end up losing precision here? If the underlying column is `DECIMAL` with a large precision, whatever `value.to_f` returns could be much lower precision.

I guess the `BigDecimal` case below has the same problem.